Source/LibationWinForms/Form1.QuickFilters.cs

```diff
@@ -56,5 +50,11 @@ private void firstFilterIsDefaultToolStripMenuItem_Click(object sender, EventArg
 		private void addQuickFilterBtn_Click(object sender, EventArgs e) => QuickFilters.Add(this.filterSearchTb.Text);
 
 		private void editQuickFiltersToolStripMenuItem_Click(object sender, EventArgs e) => new EditQuickFilters().ShowDialog();
+
+		private void productsDisplay_InitialLoaded(object sender, EventArgs e)
```

Comment:
Where does this get called?

Reply:
It's called in the designer generated code.

Reply:
Makes sense. By default the merge tool in github hides files with huge changes which always includes autogen stuff like that